library_generation/generate_library.sh

```diff
@@ -78,6 +90,10 @@ if [ -z "${include_samples}" ]; then
   include_samples="true"
 fi
 
+if [ -z "$enable_postprocessing" ]; then
+  enable_postprocessing="false"
```

Comment:
Is it better to set the default value to `true`?

Reply:
Modified